### ISSUE TITLE
__BYTE_ORDER__ check fails on gcc

### ISSUE THREAD

**Issue by hoqhuuep** [cetic/6lbr#261]
The issue is in this line:
https://github.com/cetic/6lbr/blob/develop/examples/6lbr/platform/native/6lbr-conf-native.h#L116

It should check for:
```
__BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
```

Rather than:
```
__BYTE_ORDER__ == __LITTLE_ENDIAN__
```

Source:
https://gcc.gnu.org/onlinedocs/cpp/Common-Predefined-Macros.html

I expect this is a copy paste error. The check for big endian is correct.

Thanks!

**Comment by laurentderu** [comment]
Thanks for reporting, it's fixed in dev-1.4.x and develop branches.

**Comment by hoqhuuep** [comment]
Excellent, thank you!
